core/modules/layout/layout.module

```diff
@@ -2768,3 +2768,17 @@ function _layout_clean_custom_css($classes = array()) {
 function layout_field_delete_instance($instance) {
   cache('layout_path')->flush();
 }
+
+/**
+ * Render a block
+ * @param Block $block
```

Comment:
There must be an empty line before the first `@param`. The description of the parameters must be indented.